tslint.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
I copy all rules from LeanMood front end and:
- Remove `jsx` rules and `tslint-react`
- Add ["no-angle-bracket-type-assertion": false](https://palantir.github.io/tslint/rules/no-angle-bracket-type-assertion/).

This rule does not allow casting with `<>`  because _only as type assertions work in .tsx files._ 
So, for now, it has no sense